Library/Formula/mutt.rb

```diff
@@ -72,6 +73,11 @@ class Mutt < Formula
     sha1 "94da52d50508d8951aa78ca4b073023414866be1"
   end if build.with? "confirm-attachment-patch"
 
+  patch do
+    url "http://lunar-linux.org/~tchan/mutt/patch-1.5.23.sidebar.20140412.txt"
+    sha1 "8b86f148d8e1a0b37b0b77268349f4c94e797561"
+  end if build.with? "sidebar-patch"
```

Comment:
I know the rest of the formula is styled differently, but we're migrating away from `end if build.with? "sidebar-patch"` and such.

Should be:

```ruby
if build.with? "sidebar-patch"
  patch do
    url "http://lunar-linux.org/~tchan/mutt/patch-1.5.23.sidebar.20140412.txt"
    sha256 "1234"
  end
end
```

Thanks!